pkg/datarepair/checker/checker.go

```diff
@@ -146,3 +160,51 @@ func (c *checker) offlineNodes(ctx context.Context, nodeIDs storj.NodeIDList) (o
 	}
 	return offline, nil
 }
+
+// Find invalidNodes by checking the audit results that are place in statdb
+func (c *checker) invalidNodes(ctx context.Context, nodeIDs storj.NodeIDList) (invalidNodes []int32, err error) {
+	// filter if nodeIDs have invalid pieces from auditing results
+	findInValidNodesReq := &statpb.FindInvalidNodesRequest{
+		NodeIds: nodeIDs,
+		MaxStats: &pb.NodeStats{
+			AuditSuccessRatio: 0, // TODO: update when we have stats added to statdb
+			UptimeRatio:       0, // TODO: update when we have stats added to statdb
+		},
+	}
+
+	resp, err := c.statdb.FindInvalidNodes(ctx, findInValidNodesReq)
+	if err != nil {
+		return nil, Error.New("error getting valid nodes from statdb %s", err)
+	}
+
+	// What if some valid nodes haven't been audited. Just because they don't appear doesn't mean they aren't valid
```

Comment:
remove comment. Not relevant after the FindValidNodes -> FindInvalidNodes conversion

Reply:
done